tests/test_fewsnet.py

```diff
@@ -0,0 +1,89 @@
+"""Tests for the FewsNet module."""
+import pytest
+
+from aatoolbox.datasources.fewsnet.fewsnet import download_fewsnet
+
+
+@pytest.fixture
+def _mock_download_call(mocker, tmp_path):
```

Comment:
Need to add `pytest-mock` to `requirements/test.in`

Reply:
Also I think you can make this function public, it seems to be the convention anyway, plus you have a docstring